### ISSUE TITLE
Polling for agents

### ISSUE THREAD

**Issue by vinnytroia** [byt3bl33d3r/DeathStar#65]
When starting up, deathstar powers on and "polling for agents". 
Nothing else ever happens. 

Any idea what i can try?

**Comment by dryan426** [comment]
Do you have any empire agents?

**Comment by vinnytroia** [comment]
Hi. Sorry for the long delay. I'm not sure whatt the issue was. I blew away the VM and started over. It all works now.

**Comment by vinnytroia** [comment]
I would like to re-open this. I have a new build of kali and this is happening again. I have the deathstar listener loaded in Empire. 

I manually created an agent. 

I did notice that the default URL and port did not match. One said IP:81 for the IP and had a port of 8443. 
So I made them both 81. 
But deathstar always just polls for agents. Nothing ever happens. 

I feel like I’m missing something obvious but usually it works without any setup. 

Any ideas on what I can try? My next step is to blow away the listeners and agents and just start over. 

Thoughts on where I should start looking? 

Thanks.
